extensions/amp-story/0.1/amp-story-page.js

```diff
@@ -138,6 +149,20 @@ export class AmpStoryPage extends AMP.BaseElement {
   }
 
 
+  /** @private */
+  initializeMediaPool_() {
+    const storyEl = dev().assertElement(
+        closestBySelector(this.element, 'amp-story'),
+        'amp-story-page must be a descendant of amp-story.');
+
+    storyEl.getImpl()
+        .then(storyImpl => {
+          this.mediaPoolResolveFn_(MediaPool.for(storyImpl));
+        })
+        .catch(reason => this.mediaPoolRejectFn_(reason));
```

Comment:
If this is only catching errors from `getImpl()`, you can pass this as the second param to `then()` instead of chaining a catch.

Reply:
Done